lib/authorizer/hawk.js

```diff
@@ -1,6 +1,10 @@
 var url = require('url'),
     _ = require('lodash'),
+    crypto = require('crypto'),
+    querystring = require('querystring'),
     Hawk = require('postman-request/lib/hawk'),
+    RequestBody = require('postman-collection').RequestBody,
+    urlencodedBodyBuilder = require('../requester/core-body-builder').urlencoded,
```

Comment:
bring from postman-encoder?

Reply:
Will do that while making changes related to encoding in different PR.